backuper/modules/cloud/amazon/elasticache.py

```diff
@@ -0,0 +1,99 @@
+from time import sleep
+from backuper.modules.cloud.amazon import get_amazon_client
+from backuper.utils import get_msg
+from backuper.utils.constants import amazonRegions
+from backuper.utils.validate import ValidateBase
+
+
+class ValidateElasticache(ValidateBase):
+
+    def params_validate(self, **kwargs):
```

Comment:
add engine param to parameters in action file and validate all possible parameters depends on different actions

Reply:
no need in engine param in elasticache, due to aws restrictions.